ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/application/PipelineCall.kt

```diff
@@ -15,31 +15,34 @@ import io.ktor.utils.io.*
 
 private val RECEIVE_TYPE_KEY: AttributeKey<TypeInfo> = AttributeKey("ReceiveType")
 
+@Deprecated(message = "Renamed to ServerCall", replaceWith = ReplaceWith("ServerCall"))
+public typealias ApplicationCall = ServerCall
+
 /**
  * A single act of communication between a client and server.
- * @see [io.ktor.server.request.ApplicationRequest]
- * @see [io.ktor.server.response.ApplicationResponse]
+ * @see [io.ktor.server.request.ServerRequest]
+ * @see [io.ktor.server.response.ServerResponse]
  */
-public interface ApplicationCall {
+public interface ServerCall {
     /**
      * [Attributes] attached to this call.
      */
     public val attributes: Attributes
 
     /**
-     * An [ApplicationRequest] that is a client request.
+     * An [ServerRequest] that is a client request.
      */
-    public val request: ApplicationRequest
+    public val request: ServerRequest
 
     /**
      * An [PipelineResponse] that is a server response.
      */
-    public val response: ApplicationResponse
+    public val response: ServerResponse
 
     /**
      * An application being called.
      */
-    public val application: Application
+    public val server: Server
```

Comment:
would be nice to have deprecated with `ReplaceWith` for renamed properties/functions, like this:
```suggestion
    public val server: Server
    @Deprecated("Renamed to server", ReplaceWith("server"))
    public val application: Server
```